app/scripts/modules/core/help/helpContents.js

```diff
@@ -264,6 +264,22 @@ module.exports = angular.module('spinnaker.core.help.contents', [])
     'kubernetes.containers.requests.memory': '(Optional) This is used for scheduling. It assures that this container will always be scheduled on a machine ' +
       'with at least this much memory available.',
     'kubernetes.namespace': 'The namespace you have configured with the above selected account. This will often be referred to as "Region" in Spinnaker.',
+    'kubernetes.service.ports.name': '(Optional) A name for this port. Can be found using DNS lookup if specified.',
+    'kubernetes.service.ports.port': 'The port this service will expose to resources internal to the cluster.',
+    'kubernetes.service.ports.nodePort': '(Optional) A port to open on every node in the cluster. This allows you to receive external traffic without ' +
```

Comment:
We're not consistent throughout the help contents, but Matt's suggestion is more common - field names and other labels (i.e. "Type", "Advanced Settings") should be wrapped in `<b>` tags.